qemu/tests/balloon_hotplug.py

```diff
@@ -116,9 +117,10 @@ def enable_balloon_service():
         error_context.context("Unplug balloon device for %d times" % (i+1),
                               logging.info)
 
-        out = vm.devices.simple_unplug(devs[0].get_aid(), vm.monitor)
+        out = vm.devices.simple_unplug(devs[0].get_aid(), vm.monitor,
+                                       timeout=unplug_timeout)
```

Comment:
Could you add the sleep step as we discussed in https://bugzilla.redhat.com/show_bug.cgi?id=1710669#c6?
Thanks

Reply:
@lijinlijin Updated, please help review, thanks.